viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -240,6 +243,10 @@ export class ComboControls extends EventDispatcher {
     this.triggerCameraChangeEvent();
   };
 
+  get cameraAdditionalRotation(): Quaternion {
+    return this._additionalRotation;
+  }
```

Comment:
Dafuck

Reply:
This is the key thing that got animation working. It is rotation that is applied after lookAt method was called. Value for it is set in `setCameraRotation` method and basically equal to the difference between desired rotation of the camera and the one we get after all manipulations in update function (lookAt method). So if we set rotation to the same value as the camera have right now, this additional rotation will become zero.

Reply:
So, renamed to cameraRawRotation and changed the way it is handled a little bit. (As we describe on Wednesday 16.02).